src/Microsoft.Health.Fhir.Shared.Api/Controllers/ExportController.cs

```diff
@@ -134,18 +142,26 @@ public class ExportController : Controller
             [FromQuery(Name = KnownQueryParameterNames.Container)] string containerName,
             [FromQuery(Name = KnownQueryParameterNames.TypeFilter)] string typeFilter,
             [FromQuery(Name = KnownQueryParameterNames.Format)] string formatName,
+            [FromQuery(Name = KnownQueryParameterNames.AnonymizationConfigurationLocation)] string anonymizationConfigLocation,
+            [FromQuery(Name = KnownQueryParameterNames.AnonymizationConfigurationFileEtag)] string anonymizationConfigFileETag,
             string typeParameter,
             string idParameter)
         {
             CheckIfExportIsEnabled();
 
+            if (!string.IsNullOrWhiteSpace(anonymizationConfigFileETag) || !string.IsNullOrWhiteSpace(anonymizationConfigFileETag))
+            {
+                CheckIfAnonymizedExportIsEnabled();
+                CheckContainerNameForAnonymizedExport(containerName);
+            }
+
             // Export by ResourceTypeId is supported only for Group resource type.
             if (!string.Equals(typeParameter, ResourceType.Group.ToString(), StringComparison.Ordinal) || string.IsNullOrEmpty(idParameter))
             {
                 throw new RequestNotValidException(string.Format(Resources.UnsupportedResourceType, typeParameter));
             }
 
-            return await SendExportRequest(ExportJobType.Group, since, typeFilter, resourceType, idParameter, containerName: containerName, formatName: formatName);
+            return await SendExportRequest(ExportJobType.Group, since, typeFilter, resourceType, idParameter, containerName: containerName, formatName: formatName, anonymizationConfigLocation: anonymizationConfigLocation, anonymizationConfigFileETag: anonymizationConfigFileETag);
```

Comment:
>SendExportRequest [](start = 25, length = 17)

nit: can we break this line? It's bit too big for my taste and average developer monitor

Reply:
Now separated the parameters into multiple lines.